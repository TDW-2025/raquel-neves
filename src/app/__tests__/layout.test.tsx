import { render } from "@testing-library/react";
import RootLayout from "../layout";

test("RootLayout snapshot", () => {
  const { asFragment } = render(
    <RootLayout>
      <div>Test Content</div>
    </RootLayout>,
  );
  expect(asFragment()).toMatchSnapshot();
});
