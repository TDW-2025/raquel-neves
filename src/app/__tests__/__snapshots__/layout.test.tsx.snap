// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`RootLayout snapshot 1`] = `
<DocumentFragment>
  <div>
    Test Content
  </div>
</DocumentFragment>
`;
